experimental/packages/opentelemetry-sdk-node/src/types.ts

```diff
@@ -30,8 +30,7 @@ export interface NodeSDKConfiguration {
   contextManager: ContextManager;
   defaultAttributes: SpanAttributes;
   textMapPropagator: TextMapPropagator;
-  metricProcessor: Processor;
-  metricExporter: MetricExporter;
+  metricReader: MetricReader;
   metricInterval: number;
```

Comment:
Seems like we can get rid of this option too.

Reply:
Yes, thanks for catching this. :slightly_smiling_face: 
Removed in [e376768](https://github.com/open-telemetry/opentelemetry-js/pull/2871/commits/e376768d8b398f048ca30608e4daf4bdea40ed19)